plugins/pulp_rpm/plugins/importers/yum/config_validate.py

```diff
@@ -25,3 +46,69 @@ def validate(config):
             msg += failure_message + '\n'
         msg = msg.rstrip()  # remove the trailing \n
         return False, msg
+
+
+def _validate_allow_keys(repo, config, error_messages):
```

Comment:
Imagine the situation when the exiting key was compromised, so in the update there will be a new valid key added, but the compromised one removed. If we will reject the removal of exiting key, we will end up in the situation when we will not be able to get rid of the compromised key and we will have the risk to sill accept content in repo with further syncs that were signed with the compromised key